src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -249,12 +248,13 @@ Referenced assemblies:
 - 'PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'SonarLint.VisualStudio.Core, Version=4.30.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
 - 'SonarLint.VisualStudio.Infrastructure.VS, Version=4.30.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+- 'SonarQube.Client, Version=4.30.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
```

Comment:
`SonarLint.VisualStudio.IssueVisualization` now depends on `SonarQube.Client` for the checksum calculator rather than `Core`